### PR TITLE
HCAP-1162 | Fix header alignment

### PR DESCRIPTION
**Pull request by taraepp**
Description of problem (and chosen solution): https://freshworks.atlassian.net/browse/HCAP-1162

Put horizontal scroll on overflowing container element

### CODE REVIEW

File: client/src/pages/private/ParticipantView.js [hunk 1]
```diff
@@ -30,7 +30,13 @@ export default () => {
             )}
           </CustomTabs>
         </Grid>
-        <Grid container alignItems='center' justify='flex-start' direction='column'>
+        <Grid
+          container
+          alignItems='center'
+          justify='flex-start'
+          direction='column'
+          style={{ overflowX: 'scroll' }}
```
Comment (fwpushan) [inline client/src/pages/private/ParticipantView.js:38]: try to use class instead of inline-style

**Review by fwpushan — COMMENTED** [review]
Please check my comments

File: client/src/pages/private/ParticipantView.js (hunk 1 above)
Comment (taraepp) [reply client/src/pages/private/ParticipantView.js:38]: That makes sense! I'll update that.

**Review by fwpushan — APPROVED** [review]
LGTM 👍

**Pull request merged**